File: app/src/main/java/com/example/login/addressdetail.java
package com.example.login;

public class addressdetail {

    String name,phone,address,city,state,pincode,type;

    public addressdetail() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getPincode() {
        return pincode;
    }

    public void setPincode(String pincode) {
        this.pincode = pincode;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public addressdetail(String name, String phone, String address, String city, String state, String pincode, String type) {
        this.name = name;
        this.phone = phone;
        this.address = address;
        this.city = city;
        this.state = state;
        this.pincode = pincode;
        this.type = type;
    }
}

File: app/src/main/java/com/example/login/dataclass/Addressdetail.kt
package com.example.login.dataclass

class addressdetail {
    var name: String? = null
    var phone: String? = null
    var address: String? = null
    var city: String? = null
    var state: String? = null
    var pincode: String? = null
    var type: String? = null

    constructor() {}
    constructor(name: String?, phone: String?, address: String?, city: String?, state: String?, pincode: String?, type: String?) {
        this.name = name
        this.phone = phone
        this.address = address
        this.city = city
        this.state = state
        this.pincode = pincode
        this.type = type
    }
}